core/src/main/java/hudson/triggers/Trigger.java

```diff
@@ -94,7 +94,7 @@ public void start(J project, boolean newInstance) {
         } catch (ANTLRException e) {
             // this shouldn't fail because we've already parsed stuff in the constructor,
             // so if it fails, use whatever 'tabs' that we already have.
-            LOGGER.log(Level.FINE, "Failed to parse crontab spec: "+spec,e);
+            LOGGER.log(Level.WARNING, "Failed to parse crontab spec: "+spec,e);
```

Comment:
I think we need the Item name here so you can tell from the log which job has the issue.